dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -12,13 +12,12 @@
     <!--@elvariable id="fileMetadata" type="edu.harvard.iq.dataverse.persistence.datafile.FileMetadata"-->
 
     <!-- No files accessible in Dataset -->
-    <ui:fragment rendered="#{empty datasetFilesTab.fileMetadatasSearch.getAllAvailableFilesCount() and datasetFilesTab.dataset.hasEverBeenPublished()
+    <ui:fragment rendered="#{datasetFilesTab.fileMetadatasSearch.getAllAvailableFilesCount() eq 0 and datasetFilesTab.dataset.hasEverBeenPublished()
                             and !datasetFilesTab.isFilesTabDisplayable()}">
         <div class="panel panel-default" jsf:rendered="#{!settingsWrapper.rsyncOnly}">
             <div class="panel-body">
                 <div class="col-sm-9">
-                    <ui:fragment rendered="#{!datasetFilesTab.dataset.hasActiveEmbargo()}">
-                        <p>#{bundle['file.notFound.tip']}</p>
+                    <ui:fragment rendered="#{!datasetFilesTab.dataset.hasActiveEmbargo()}">                        <p>#{bundle['file.notFound.tip']}</p>
```

Comment:
Maybe it would be better if we would leave `<p>` tag formatting as it was before?

Reply:
Some auto-formatting happened I think.